src/scale.ts

```diff
@@ -273,6 +275,7 @@ export function scaleTypeSupportProperty(scaleType: ScaleType, propName: string)
     case 'domain':
     case 'range':
     case 'scheme':
+    case 'interpolate':
```

Comment:
Hint: Try if this works in all quantitative domain scales.  

(Linear should definitely work.  I also just recalled that you also haven't help me tested if it works for sequential in Vega -- I guess it should work and might be a bug.)